README.rst

```diff
@@ -6,13 +6,166 @@ App for tracking sprint commitments, vacations and spillovers.
+
+
+Completing the sprints
+~~~~~~~~~~~~~~~~~~~~~~
+In case when users needs to schedule tickets for sprints that haven’t been created yet, they can press the `Create Next Sprint` to create a new one for the currently viewed cell.
+
+Completing the sprints
+~~~~~~~~~~~~~~~~~~~~~~
+To complete the sprint, you need to have `Staff status` permissions.
+The main idea behind this is that they are not shared by cells - you need to have separate sprint for each one. You can press the `Complete Sprint` button on the cell's dashboard to schedule a Celery task with the following pipeline:
+
+1. Upload spillovers.
+    This uploads all spillovers to the `GOOGLE_SPILLOVER_SPREADSHEET`. The following rows are filled in the spreadsheet:
+
+    a) Ticket
+        The key of the ticket.
+    b) Status
+        The status of the ticket at the moment of ending the sprint.
+    c) Sprints
+        The active sprint (the one that is currently being ended).
+    d) Assignee
+        The assignee, for whom the spillover is being counted.
+    e) Reviewer 1
+    f) Reviewer 2
+    g) Reporter
+    h) Story Points
+    i) Estimated time
+        The initial estimation of the ticket (in hours).
+    j) Remaining time
+        The remaining time for the ticket (in hours).
+    k) Reason for the spillover
+        The reason of the spillover is retrieved from the comments made within the active sprint. The assignees should provide it with a comment matching the following regexp: ```[~{JIRA_BOT_USERNAME}\]: <spillover>(.*)<\/spillover>```. In case of multiple occurrences of comments matching this regexp, only the last one is taken into account. In case of no occurrences of such comments, the Jira bot will create a comment defined in `SPILLOVER_REMINDER_MESSAGE`.
+
+    If the users have achieved the clean sprint (without spillovers), they can post some hints on the ticket with the `SPRINT_MEETINGS_TICKET` name by adding a comment matching the spillover reason regexp (provided above). In case of no such comment, they will be reminded on the ticket with `SPILLOVER_CLEAN_HINTS_MESSAGE` comment. It's possible to disable the pings for specific users by adding them to `SPILLOVER_CLEAN_SPRINT_IGNORED_USERS` (this can be useful for people that are members of multiple cells, as they will be pinged on each cell-specific ticket).
+2. Upload commitments.
+    The `goal` of each user from the dashboard is uploaded to the cell-specific commitments sheet of the `GOOGLE_SPILLOVER_SPREADSHEET`.
+3. Move archived issues out of the active sprint.
+    There has been a bug before that disallowed completing the sprint if it had archived issues, so we're moving all of them out of the active sprint.
+4. Close the active sprint.
+5. Move issues from the closed sprint to the next one.
+6. Open the next sprint.
+7. Create role-specific tasks for the future sprint.
```

Comment:
@Agrendalath It might make sense to replace "future sprint" with "next sprint" here, to make it clear that these tickets are being created for the "next sprint" that was opened in the previous step.

Reply:
If we're closing sprint N, they are being created for the sprint N+2, hence such naming.

Should we indicate it in a different way to make this clear?

Reply:
@Agrendalath Ah OK! Yep, how about:

> Create role-specific tasks for the sprint after next.

Reply:
Sound good!